cnames_active.js

```diff
@@ -681,6 +681,7 @@ var cnames_active = {
   "docsify-es": "sidval.github.io/docsify-es",
   "docsify-ru": "truepatch.github.io/docsify-ru",
   "docsite": "txd-team.github.io/docsite-doc-v1",
+  "docs.quickonline": "d71413bb-114b-44b9-88e2-464e035736b2.repl.co",
```

Comment:
```suggestion
  "docs.quickonline": "d71413bb-114b-44b9-88e2-464e035736b2.repl.co", //  noCF
```
As this is a sub-sub-domain, it cannot use Cloudflare's proxy/cache/SSL, so the `// noCF` comment must be added.

Reply:
ok